src/test/fixtures/mocks/request-animation-frame.js

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+export default function mockRaf() {
+  let fns = [];
+  jest.spyOn(window, 'requestAnimationFrame').mockImplementation(fn => {
+    fns.push(fn);
+  });
+
+  return () => {
+    for (const fn of fns) {
+      fn();
+    }
+    fns = [];
+  };
```

Comment:
Wondering if you could use https://facebook.github.io/jest/docs/en/mock-function-api.html#mockfnmockcalls instead of your own array. Something like:

```javascript
const spy = jest.spyOn(window, 'requestAnimationFrame').mockImplementation(() => {});
return () => {
  for (const [ fn ] of spy.calls) {
    fn();
  }
  spy.mockClear(); // to make it possible to call the function several times.
};
```